Containers/Login/Login.js

```diff
@@ -1,39 +1,92 @@
+                            value={this.state.password}
+                        />
+                        <TouchableOpacity style={{ backgroundColor: '#e9e9e9', width: 200, padding: 10, borderRadius: 15 }}>
+                            <Text style={{textAlign: 'center'}}>Log in</Text>
+                        </TouchableOpacity>
+                    </View> 
                 </ImageBackground>
             </View>
-        )
+            )
+            } else {
+                return (
+                    <View style={styles.container}>
+                    <ImageBackground source={require('../../assets/goldengate.jpg')} style={{width: '100%', height: '100%', display: 'flex', flex: 1, flexDirection: 'column', justifyContent: 'space-evenly', alignItems: 'center'}}>
+                    <View style={styles.overlay} />
+                        <Icon color="white" name="user-circle" type="font-awesome" size={70} padding={15} top={50}/>
+                            <View>
+                            <TextInput
+                                placeholder='Username'
+                                style={{height: 40, width: 200, borderColor: 'gray', borderWidth: 2, backgroundColor: 'white', opacity: 0.7, padding: 5, textAlign: 'center', top: -20}}
+                                onChangeText={(username) => this.setState({username})}
+                                value={this.state.username}
+                            />
+                            <TextInput
+                                placeholder='Password'
+                                secureTextEntry={true} 
+                                password={true}
+                                style={{height: 40, width: 200, borderColor: 'gray', borderWidth: 2, backgroundColor: 'white', opacity: 0.7, top: -10, padding: 5, textAlign: 'center'}}
+                                onChangeText={(password) => this.setState({password})}
+                                value={this.state.password}
+                            />
+                            <TextInput
+                                placeholder='Confirm Password'
+                                secureTextEntry={true} 
+                                password={true}
+                                style={{height: 40, width: 200, borderColor: 'gray', borderWidth: 2, backgroundColor: 'white', opacity: 0.7, padding: 5, textAlign: 'center'}}
+                                onChangeText={(confirmPassword) => this.setState({confirmPassword})}
+                                value={this.state.confirmPassword}
+                            />
+                        <TouchableOpacity style={{ backgroundColor: '#e9e9e9', width: 200, padding: 10, borderRadius: 15, top: 10 }}>
+                            <Text>Create account</Text>
+                        </TouchableOpacity>
+                    </View> 
+                    </ImageBackground>
+                </View>
+                )
+            }
+        }
     }
```

Comment:
We could possibly refactor this

Reply:
If they are logged in there is a different button. If they aren't logged in, there is a different button and 1 extra input field for confirming their password